backend/cfg/cfg_with_layout.ml

```diff
@@ -83,6 +83,22 @@ let assign_blocks_to_section t labels name =
 
 let get_section t label = Hashtbl.find_opt t.sections label
 
+let merge_sections t label other =
```

Comment:
What is the intended semantics of
this operation? For instance, I first
expected it to me symmetric, but it
looks like `None, Some ""` and
`Some "", None` do not have the
same effect.

(It might be clearer by simultaneously
matching on both sections.)

Reply:
I _think_ that we should never have both a `None` and a `Some`, so it is not supposed to matter what happens in those cases.